src/apps/gateway/videos/index.test.ts

```
@@ -1,16 +1,10 @@
 import { validateRequest } from 'src/utils/validator';
-import { logger } from 'src/utils/logger';
-import { createCloudTasks } from '../../../utils/cloud-task';
-import { verifySignature } from '../../../services/videos/convert/validator';
+import { streamToStorage } from './routes/stream-to-storage';
+import { ConvertSchema } from '../../../services/videos/convert/schema';
 
 // Mock dependencies
-// let routeHandlers: { path: string; middlewares: Function[] }[] = [];
-type RouteHandler = (req: Request, res: Response, next: NextFunction) => void;
-let routeHandlers: { path: string; middlewares: RouteHandler[] }[] = [];
+let routeHandlers: { path: string; middlewares: Function[] }[] = [];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)